lib/index.js

```diff
@@ -302,10 +302,10 @@ module.exports = mergeExports(fn, {
 	// TODO webpack 7 remove
 	get WebpackOptionsValidationError() {
 		return util.deprecate(
-			require("schema-utils").ValidationError,
+			() => require("schema-utils").ValidationError,
```

Comment:
`util.deprecate` aworks with classes

Reply:
but you only get the deprecation message when calling `new WebpackOptionsValidationError`, `instanceof WebpackOptionsValidationError` would still work fine without triggering the warning